docs/defi/fcm/math.md

```diff
@@ -0,0 +1,705 @@
+
+### Effective Collateral with Price Correlation
+
+When collateral types are correlated (e.g., FLOW and stFLOW):
+
+**Simplified (no correlation)**:
+```
+Risk = \sum_i Risk_i
+```
+
+**With correlation** (advanced):
+```
+Risk = \sqrt{\sum_i\sum_j w_i w_j \sigma_i \sigma_j \rho_ij}
+```
+
+Where:
+- $w_i$: Weight of asset $i$
+- $\sigma_i$: Volatility of asset $i$
+- $\rho_ij$: Correlation between assets $i$ and $j$
+
+**Practical impact**:
+```
+Scenario 1: Uncorrelated collateral
+- 50% FLOW (volatile)
+- 50% USDC (stable)
+- Effective diversification
+
+Scenario 2: Correlated collateral
+- 50% FLOW (volatile)
+- 50% stFLOW (volatile, correlated with FLOW)
+- Limited diversification
+- Both can drop together!
+```
+
+## Yield Calculations
+
+### Simple APY
+
+Annual Percentage Yield without compounding:
+
+```
+APY_simple = (FinalValue - InitialValue / InitialValue) × (365 / Days)
+```
+
+### Compound APY
+
+With continuous compounding:
+
+```
+APY_compound = e^r - 1
+```
+
+Where $r$ is the continuous annual rate.
+
+### Leveraged Yield
+
+When borrowing to increase yield exposure:
+
+```
+Yield_leveraged = Yield_strategy - Interest_borrowed
+```
+
+**Example**:
+```
+Deposit: $1000 collateral
+Borrow: $615 at 5% APY
+Deploy $615 to strategy earning 10% APY
+
+Costs:
+- Interest on borrowed: 615 × 0.05 = $30.75/year
+
+Returns:
+- Yield from strategy: 615 × 0.10 = $61.50/year
+
+Net leveraged yield: 61.50 - 30.75 = $30.75/year
+Effective APY on your $1000: 30.75 / 1000 = 3.075% extra
+Total return: Base yield + leveraged yield
+```
+
+## Risk Metrics
+
+### Liquidation Risk Score
+
+A simplified risk score:
+
+```
+\text{Risk Score} = (1 / HF - 1.0) × Volatility_collateral
+```
+
+Higher score = higher risk.
+
+### Value at Risk (VaR)
+
+Maximum expected loss over time period at confidence level:
+
+```
+VaR_95% = EC × \sigma × z_0.95
+```
+
+Where:
+- $\sigma$: Daily volatility of collateral
+- $z_0.95$: Z-score for 95% confidence (≈1.645)
+
+**Example**:
+```
+Collateral: $1000 FLOW
+Daily volatility: 5%
+Confidence: 95%
+
+VaR = 1000 × 0.05 × 1.645 = $82.25
+
+Interpretation: 95% confident that daily loss won't exceed $82.25
+```
+
+## Validation & Safety Checks
+
+### Health Factor Bounds
+
+All operations must satisfy:
+
+```
+1.0 ≤ HF_min < HF_target < HF_max
+```
+
+Typical values: $HF_min = 1.1$, $HF_target = 1.3$, $HF_max = 1.5$
+
+### Collateral Factor Bounds
+
+For safety:
+
+```
+0 < CF_t ≤ 1.0
+```
+
+Typically:
+- Volatile assets (FLOW): $CF = 0.75 - 0.85$
+- Stable assets (USDC): $CF = 0.90 - 0.95$
+- Liquid staking (stFLOW): $CF = 0.80 - 0.85$
+
+### Maximum Leverage
+
+Maximum theoretical leverage:
+
+```
+MaxLeverage = (1 / 1 - CF)
+```
+
+**Examples**:
+```
+CF = 0.8: Max leverage = 1 / (1 - 0.8) = 5x
+CF = 0.75: Max leverage = 1 / (1 - 0.75) = 4x
+CF = 0.9: Max leverage = 1 / (1 - 0.9) = 10x (risky!)
+```
+
+But actual safe leverage is constrained by target health:
+
+```
+SafeLeverage = (CF / HF_target)
```

Comment:
Wondering if we should rename this to Debt Ratio as leverage is typically = total exposure/equity. Leverage is typically = C/(C-D). If D = C * (CF)/(HF) then the leverage would be 1/(1-(CF/HF)) = 1/(1 - 0.615) = 2.6x